user-guide/Advanced_Modules/User-Defined_APIs/Defining_an_API/UD_APIs_Defining_a_new_API.md

```diff
@@ -99,6 +104,24 @@ There are two ways to pass data to the API script if you make use of the *OnApiT
 > [!TIP]
 > To see how these two options are used in an API script, see [API script examples](xref:UD_APIs_API_script_examples).
 
+##### Query parameters
```

Comment:
Maybe move above the request body since we also mention it first in the introduction text?